runners/gearpump/pom.xml

```diff
@@ -0,0 +1,197 @@
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.beam</groupId>
+        <artifactId>runners-parent</artifactId>
+        <version>0.1.0-incubating-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>gearpump</artifactId>
+
+    <name>Apache Beam :: Runners :: Gearpump</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <java.version>1.7</java.version>
+        <gearpump.version>0.8.1-SNAPSHOT</gearpump.version>
+        <beam.version>0.1.0-incubating-SNAPSHOT</beam.version>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>apache.snapshots</id>
+            <name>Apache Development Snapshot Repository</name>
+            <url>https://repository.apache.org/content/repositories/snapshots/</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>gearpump-shaded-repo</id>
+            <name>Vincent at Bintray</name>
+            <url>http://dl.bintray.com/fvunicorn/maven</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.gearpump</groupId>
+            <artifactId>gearpump-streaming_2.11</artifactId>
+            <version>${gearpump.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.gearpump</groupId>
+            <artifactId>gearpump-daemon_2.11</artifactId>
+            <version>${gearpump.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.gearpump</groupId>
+                    <artifactId>gearpump-experimental-cgroup_2.11</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.beam</groupId>
+            <artifactId>java-sdk-all</artifactId>
+            <version>${beam.version}</version>
```

Comment:
I believe you can omit this version and let the parent `<dependencyManagement>` sections set it up.